### PR TITLE
NAKIJIN-230. Add an endpoint to move the locality candidates to the locality synonyms

### PR DESCRIPTION
**Pull request by oleg-n-k**
Add an endpoint to move the locality candidates to the locality synonyms

### CODE REVIEW

**Review by DFSko — APPROVED** [review]

File: HappyTravel.Nakijin.Api/Services/AdministratorServices/LocalityCandidatesManagementService.cs [hunk 1]
```diff
@@ -58,24 +55,88 @@ async Task<Result<List<LocalityCandidate>>> CheckIfLocalitiesDoNotExist(List<Loc
             }
 
 
-            Task RemoveLocalityCandidates(List<LocalityCandidate> localityCandidates)
+            Task AddLocalityCandidates(List<LocalityCandidate> localityCandidates)
```
Comment (hasmikharutyunyan) [inline HappyTravel.Nakijin.Api/Services/AdministratorServices/LocalityCandidatesManagementService.cs:58]: maybe `AddLocalityCanditatesToLocalities` or `AddLocalities`

File: HappyTravel.Nakijin.Data/Models/LocalitySynonym.cs [hunk 2]
```diff
@@ -5,10 +5,11 @@ namespace HappyTravel.Nakijin.Data.Models
 {
     public class LocalitySynonym
     {
-        public int Id { get; set; }
-        public MultiLanguage<string> Names { set; get; }
-        public DateTime Created { get; set; }
-        public int LocalityId { get; set; }
+        public int Id { get; init; }
+        public MultiLanguage<string> Names { get; init; }
```
Comment (hasmikharutyunyan) [inline HappyTravel.Nakijin.Data/Models/LocalitySynonym.cs:9]: I think synonym can have only default name (in english)

File: HappyTravel.Nakijin.Data/Models/LocalitySynonym.cs [hunk 3]
```diff
@@ -5,10 +5,11 @@ namespace HappyTravel.Nakijin.Data.Models
...
+        public DateTime Created { get; init; }
+        public DateTime Modified { get; init; }
+        public int LocalityId { get; init; }
```
Comment (hasmikharutyunyan) [inline HappyTravel.Nakijin.Data/Models/LocalitySynonym.cs:12]: I think LocalityId can be nullable

File: HappyTravel.Nakijin.Api/Models/LocalitySynonyms/AddLocalitySynonymsRequest.cs [hunk 4]
```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace HappyTravel.Nakijin.Api.Models.LocalitySynonyms
+{
+    public record AddLocalitySynonymsRequest
```
[the remark this answers is not in the document]
Comment (oleg-n-k) [reply HappyTravel.Nakijin.Api/Models/LocalitySynonyms/AddLocalitySynonymsRequest.cs:5]: Ok, fixed

File: HappyTravel.Nakijin.Api/Services/AdministratorServices/LocalityCandidatesManagementService.cs (hunk 1 above)
Comment (oleg-n-k) [reply HappyTravel.Nakijin.Api/Services/AdministratorServices/LocalityCandidatesManagementService.cs:58]: fixed

File: HappyTravel.Nakijin.Data/Models/LocalitySynonym.cs (hunk 2 above)
Comment (oleg-n-k) [reply HappyTravel.Nakijin.Data/Models/LocalitySynonym.cs:9]: discussed

**Review by hasmikharutyunyan — APPROVED** [review]

**Pull request merged**